config/src/main/resources/registry.conf

```diff
@@ -1,12 +1,17 @@
 registry {
```

Comment:
Should we rename this to `default_refistry.conf`?

Reply:
It will be refactored in a future release, and if it is renamed now it will result in a low version incompatibility issue.